### DISCUSSION TITLE
Minor Typo - FAQ SQL Server Maintenance Solution

### DISCUSSION THREAD

**Discussion by spjeff** [olahallengren/sql-server-maintenance-solution#469] [General] [answered]
Page has "the the number of hours" with "the" repeated twice on step # 3 at https://ola.hallengren.com/frequently-asked-questions.html

Thank you@olahallengren for sharing these great scripts.  Keep up good work.

**Comment by olahallengren** [comment, marked as answer]
This issue has been fixed. Thank you for reporting it.

**Reply by spjeff** [reply]
Thank you Ola.   Excellent script.  Recommend to everyone and great contribution to the community.
